package ru.goodbadnews.rest.exceptions;

public class UsernameFoundException extends RuntimeException {
}
